doc/userguide/support-status.rst

```diff
@@ -0,0 +1,490 @@
+Support Status
+==============
+
+Levels of Support
+-----------------
+
+Tier 1
+~~~~~~
+
+Core team develops and supports. Compiler errors or functional
```

Comment:
wonder if we should `s/Core team/OISF/g` for the entire doc

Reply:
Since we don't have two different teams within OISF I would agree or at least add OISF so either `OISF team` or `OISF core team`

Reply:
I think I would go with OISF team, as `OISF core team` might be interpreted as "there is a non-core OISF team"...

Reply:
Trying simply `OISF`.